contribs/drt/src/main/java/org/matsim/contrib/drt/optimizer/VehicleDataEntryFactoryImpl.java

```diff
@@ -136,7 +134,7 @@ public boolean isNotEligibleForRequestInsertion(DvrpVehicle vehicle, double curr
 	}
 
 	static double calcVehicleSlackTime(DvrpVehicle vehicle, double now) {
-		DrtStayTask lastTask = (DrtStayTask)Schedules.getLastTask(vehicle.getSchedule());
+		DefaultStayTask lastTask = (DefaultStayTask)Schedules.getLastTask(vehicle.getSchedule());
```

Comment:
👍 

Whoops. Some leftover from the old design (prior to adding interfaces). Now I see there are a few other places like this where the interfaces should be used instead. I will handle that in a separate PR.

Reply:
👍